ghost/core/test/e2e-api/admin/__snapshots__/members-newsletters.test.js.snap

```diff
@@ -63,7 +63,7 @@ exports[`Members API - With Newsletters - compat mode Can fetch members who are
 Object {
   "access-control-allow-origin": "http://127.0.0.1:2369",
   "cache-control": "no-cache, private, no-store, must-revalidate, max-stale=0, post-check=0, pre-check=0",
-  "content-length": "2056",
```

Comment:
How to tell there is no mapper in place 😂

Reply:
😄 Yeah that bugged me as well. I have marked it in my todos to have a look towards the end of Free trials if time permits to sort out